lib/run-compilation.js

```diff
@@ -45,9 +40,15 @@ module.exports = function runCompilation(options, compilation, done) {
       console.log(chalk.red(err));
     });
 
-  // eslint-disable-next-line no-unused-expressions
-  compilation.plugin && compilation.plugin('compilation', function (compilation) {
-    compilation.errors = compilation.errors.concat(errors);
-    compilation.warnings = compilation.warnings.concat(warnings);
+  compiler.plugin('compilation', function (compilation) {
```

Comment:
Sorry for the delayed response @karlhorky - 

Please make this a named function [because if somehow there's some error thrown it will provide better feedback in the stack trace about where it's coming from]. I should really have a lint rule for this.